incubator/solr/solr.yaml

```diff
@@ -0,0 +1,60 @@
+#
+# Docker usage: https://hub.docker.com/_/solr/
+#
+# A headless service to create DNS records
+apiVersion: v1
+kind: Service
+metadata:
+  name: solr
+  labels:
+    app: solr
+spec:
+  ports:
+  - port: 8983
+    protocol: TCP
+    name: solr
+  # solr-*.solr.default.svc.cluster.local
+  clusterIP: None
+  selector:
+    app: solr
+---
+apiVersion: apps/v1alpha1
+kind: PetSet
+metadata:
+  name: solr
+spec:
+  serviceName: "solr"
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: solr
+      annotations:
+        pod.alpha.kubernetes.io/initialized: "true"
+    spec:
+      terminationGracePeriodSeconds: 0
+      containers:
+      - name: solr
```

Comment:
We need http://kubernetes.io/docs/user-guide/compute-resources/ defined.  And will need Java Heap settings here as well.  What settings does the container allow for?  What settings does `/opt/solr/bin/solar` pick up?  How much memory should we launch with?